src/Tribe/Customizer.php

```diff
@@ -583,7 +583,7 @@ public function register( WP_Customize_Manager $customizer ) {
 			 * Allows people to Register and de-register the method to register more Fields
 			 *
 			 * @since 4.4
-			 * @since TBD Add Customizer instance as a parameter.
+			 * @since 4.12.15 Add Customizer instance as a parameter.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: ae4249caa84eb5f453cc0c656db978c6</sub>